src/lib/createHelpers.ts

```diff
@@ -8,16 +8,22 @@ import {
 
 type HoganRenderer = (value: any) => string;
 
+interface HoganHelpers {
```

Comment:
Was the ESLint configuration complaining about the structure not being created prior to usage here?

Reply:
The ESLint configuration wants us to explicitly describe the return type of the `hoganHelpers` function. I could have put it inline without creating an interface but I found it clearer that way. 

Rule: [explicit-function-return-type](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md)